### PR TITLE
API extensions

### PR DESCRIPTION
**Pull request by kiliit**
Some extensions on the API

### CODE REVIEW

**Comment by kiliit** [comment]
Notice that the dependency from gogits/go-gogs-client is changed to kiliit/go-gogs-client

**Comment by tboerger** [comment]
To avoid merge commits you should use rebase and force push your pull request ;)

File: cmd/web.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ import (
 	"gopkg.in/ini.v1"
 	"gopkg.in/macaron.v1"
 
-	api "github.com/gogits/go-gogs-client"
+	api "github.com/kiliit/go-gogs-client"
```
Comment (Unknwon) [inline cmd/web.go]: Maybe you want re-PR this after api client is merged. :sweat:

Comment (kiliit) [inline cmd/web.go]: I can make one more PR, rebase and force push if it makes you :sweat_smile:

Comment (Unknwon) [inline cmd/web.go]: Yes, please do that.

**Pull request closed without merging**